packages/server/src/socket/service.ts

```diff
@@ -21,6 +21,19 @@ export class SocketService extends Service {
     this.cacheByUserId = await this.cachingService.newServerCache('cache_sockets_by_user_id')
   }
 
+  async destroy() {
+    for (const sockets of Object.values(this.socketsByUserId)) {
+      if (!sockets) {
+        continue
+      }
+
+      for (const socket of sockets) {
+        // TODO: Delete the socket from the cache and invalidate once we distribute the socket usage
+        socket.disconnect()
+      }
+    }
+  }
```

Comment:
You can use `this.sockets` instead to list all sockets.

Reply:
This function was removed as it is unnecessary!